### DISCUSSION TITLE
Github student<title>

### DISCUSSION THREAD

**Discussion by helloLari** [community/community#39687] [GitHub Education]
### Select Topic Area

Bug

### Body

Querida equipe,
Tentei continuar minha inscrição no Github Education, mas falha em diferentes computadores e navegadores.
Após carregar minha carteira de estudante, ele diz que não pode acessar minha localização e, portanto, não pode prosseguir com a solicitação.
No entanto, o site não pede para acessar minha localização, não tenho a chance de permitir.
Em vez disso, sempre que a webcam for solicitada.
O problema permanece o mesmo se eu tentar fotografar a webcam em vez de carregar minha carteira de estudante.
Verifiquei novamente as configurações do meu navegador e, de fato, as solicitações de localização são permitidas.
Eu tentei isso no meu local de residência, bem como na universidade, em um computador desktop, no Firefox e no Chrome.
Obrigado por qualquer ajuda!

**Comment by ossanna16** [comment]
Hi there @helloLari and welcome to our community! Thank you for reporting this bug!

**Reply by helloLari** [reply]
É possível renovar a inscrição no Github Education?
